src/bmark/bmark_solvers.jl

```diff
@@ -1,4 +1,7 @@
-export bmark_solvers, profile_solvers, bmark_and_profile
+export bmark_solvers, profile_solvers, profile_solvers_by_time,
+       profile_solvers_by_iterations, profile_solvers_by_evaluations,
+       bmark_and_profile, bmark_and_profile_by_time,
+       bmark_and_profile_by_iterations, bmark_and_profile_by_evaluations
```

Comment:
We should be smarter here and record stats in some sort of Dict indexed by a symbol (`:eval`, `:time`, etc.) instead of multiplying the functions.